system/src/commonMain/kotlin/com/splendo/kaluga/system/network/state/NetworkState.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.system.network.state
+
+import com.splendo.kaluga.state.State
+import com.splendo.kaluga.system.network.Network
+import com.splendo.kaluga.system.network.BaseNetworkManager
+
+sealed class NetworkState(
+    open val networkType: Network,
+    open val networkManager: BaseNetworkManager
```

Comment:
remove in favor of comment at https://github.com/splendo/kaluga/pull/197/files#diff-081d6e4a2ebb3806d331a8856f6cd33ef670da9e3af8dcf03d525c84b6b625c3R42

Reply:
3c6faf9